packages/portal-proto/src/components/Table/VerticalTable.tsx

```diff
@@ -267,11 +305,15 @@ function VerticalTable<TData>({
                         }
                         colSpan={header.colSpan}
                       >
-                        <button className="flex items-center font-heading">
-                          {flexRender(
-                            header.column.columnDef.header,
-                            header.getContext(),
+                        <button
+                          className="flex items-center font-heading"
+                          onClick={handleSorting}
+                          onKeyDown={createKeyboardAccessibleFunction(
+                            handleSorting,
                           )}
+                          aria-label={ariaText}
```

Comment:
aria-label can be used on buttons, it is specifically for interactive elements like buttons where there is not a visible label: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label.

I do agree that this button should not have an aria-label because it already has a label in the form of the button's content.